cli/internal/packager/inspect.go

```diff
@@ -32,7 +32,7 @@ func Inspect(packageName string) {
 	utils.ColorPrintYAML(text)
 
 	// Load the config to get the build version
-	config.LoadConfig(tempPath.base + "/zarf.yaml")
+	_ = config.LoadConfig(tempPath.base + "/zarf.yaml")
```

Comment:
We should handle an error here or they could get a crash

Reply:
copied what inspect.go does on an error loading config